docs/development/local_setup_en.md

```diff
@@ -23,3 +23,14 @@ zip -r swagger-ui-4.9.0-1.zip dist
 sha256sum swagger-ui-4.9.0-1.zip > swagger-ui-4.9.0-1.zip.sha256
 3. add the artefacts to the release https://github.com/cloudogu/swagger-ui/releases
 ```
+
+### Setup since 5.17.14-1
+Releases can be found on the official swagger-ui download page.
+The fork will be dismissed.
+```
+1. download artefact from swagger-ui
+2. generate the checksum 
+sha256sum swagger-ui.zip
+3. Edit Dockerfile for new version
+```
```

Comment:
This content does not seem like a literal translation of the german variant. Why the difference?